vue-frontend/src/components/contributions/AnimatedCreation.vue

```diff
@@ -43,7 +43,7 @@
       <div class="tw-flex tw-flex-col tw-gap-8 tw-w-[50%]">
         <div v-for="animation in animations.slice(0, 3)" :key="animation.id">
           <div
-            class="tw-relative tw-group tw-cursor-pointer"
+            class="tw-relative tw-group tw-cursor-pointer tw-overflow-hidden"
             @click="openBlog(animation)"
```

Comment:
I think this section not contaning any blog

Reply:
yes, its blog only.